tests/test_lift.py

```diff
@@ -103,3 +95,63 @@ def test_process_pallets_handles_ship_exception(self):
         lift.process_pallets([pallet])
 
         self.assertEqual(pallet.success, (False, 'ship error'))
+
+    @patch('shutil.rmtree')
+    @patch('shutil.copytree')
+    def test_copy_data(self, copytree_mock, rmtree_mock):
+        three = 'C:\\MapData\\three.gdb'
+        two = 'C:\\MapData\\two.gdb'
+
+        class CopyPalletOne(Pallet):
+            def __init__(self):
+                super(CopyPalletOne, self).__init__()
+
+                self.copy_data = ['C:\\MapData\\one.gdb', two]
+
+        class CopyPalletTwo(Pallet):
+            def __init__(self):
+                super(CopyPalletTwo, self).__init__()
+
+                self.copy_data = ['C:\\MapData\\one.gdb', three]
+
+        class CopyPalletThree(Pallet):
+            def __init__(self):
+                super(CopyPalletThree, self).__init__()
+
+                self.copy_data = ['C:\\MapData\\four.gdb', three]
+        palletThree = CopyPalletThree()
+        palletThree.is_ready_to_ship = Mock(return_value=False)
+
+        lift.copy_data([CopyPalletOne(), CopyPalletTwo(), palletThree], ['dest1', 'dest2'])
+
+        self.assertEqual(copytree_mock.call_count, 6)
```

Comment:
should these counts be 3? CopyPallet 1 and 2 have one.gdb in common and two and three are unique. CopyPallet3 isn't ready to ship so it should not have copy called?

So copytree_mock would get called once for one.gdb and again for two.gdb and three.gdb?